lib/render/editor/editor.html

```diff
@@ -459,13 +533,15 @@
           .then((results) => {
             if (promise !== this.serverPromise) return;
 
-            this._onRemoteExecutionStop(results)
+            this._onRemoteExecutionStop(results);
           })
           .catch(() => {
             mumuki.kids.showResult({status: 'aborted'})
           });
         this.serverPromise = promise;
 
+        editor.toggleInteractiveMode();
```

Comment:
Should we call this only if `editor.interactiveMode`?

Reply:
I could add an if here. It doesn't affect the current layout, as it is just toggling .play-mode that doesn't change anything without .mu-kids-interactive